test/conformance/testdata/conformance.yaml

```diff
@@ -1234,6 +1234,15 @@
     StorageClass or a dynamic provisioner.
   release: v1.9
   file: test/e2e/apps/statefulset.go
+- testname: PodTemplate lifecycle
+  codename: '[sig-architecture] PodTemplates should run the lifecycle of PodTemplates
```

Comment:
This should be [sig-node], you should regenerate this file

Reply:
Updated in ce97b499652